From a71ed1adc0278ee4a3cff78541ceab9128960d7d Mon Sep 17 00:00:00 2001
From: lcwang <lcwang@ieiworld.com>
Date: Mon, 16 Jun 2025 11:59:58 +0800
Subject: [PATCH 1/1] arm64: dts: rk3588-b675: disable mipi dsi and lvds
 displays

This is a workaround to set the hdmi display
as the default primary display.
---
 arch/arm64/boot/dts/rockchip/rk3588-b675.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588-b675.dts b/arch/arm64/boot/dts/rockchip/rk3588-b675.dts
index caf79a6b1505..5cb878dd2a2b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-b675.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-b675.dts
@@ -342,7 +342,7 @@ &combphy2_psu {
  * when dsi0 is enabled
  */
 &dsi0 {
-	status = "okay";
+	status = "disabled";
 };
 
 &dsi0_in_vp2 {
@@ -655,7 +655,7 @@ &pwm13 {
  * when dsi1 is enabled
  */
 &dsi1 {
-	status = "okay";
+	status = "disabled";
 };
 
 &dsi1_in_vp2 {
@@ -1000,11 +1000,11 @@ rgmii_phy1: phy@1 {
 };
 
 &mipi_dcphy0 {
-	status = "okay";
+	status = "disabled";
 };
 
 &mipi_dcphy1 {
-	status = "okay";
+	status = "disabled";
 };
 
 &sata2 {
-- 
2.34.1

